src/main/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+spring:
+    profiles: test
+updateTriggeringEndpoint: /send-update
+updateDestinationEndpoint: /topic/myscores
+environment: test
+servers:
```

Comment:
Why is it here? What it will finally be used for?

Reply:
This is me trying to be a little more descriptive and show some examples. xD 

This doesn't have any purpose for our app, as well as separate configuration for 'test' and 'production' environment, since we don't have any long running infrastructure.

But if I was to cut this and deliver only stuff I actually need, this would be really vague in my opinion, this is why I left this for the time being. Do you want me to delete everything which is not needed?